addons/web/static/src/views/form/form_controller.scss

```diff
@@ -447,51 +288,8 @@ $o-form-label-margin-right: 0px;
 
     // Groups
     .o_group {
-        display: inline-block;
-        width: 100%;
-        margin: 10px 0;
-
-        .o_group {
-            margin: 0;
-        }
-
-        // o_group contains nested groups
-        @for $i from 1 through $o-form-group-cols {
-            .o_group_col_#{$i} {
-                display: inline-block;
-                width: floor(100% / 12 * $i);
-                vertical-align: top;
-            }
-        }
-
-        &.o_inner_group {
-            display: inline-table;
-
-            > tbody > tr > td {
-                &.o_td_label {
-                    @include o-td-label-style;
-                }
-                vertical-align: top;
-
-                span, .o_field_boolean, .oe_avatar, .o_form_uri {
-                    &.o_field_widget {
-                        width: auto;
-                    }
-                }
-            }
-
-            .o_field_widget {
-                &.o_field_many2one_avatar {
-                    .o_field_many2one_selection {
-                        width: calc(100% - 24px);
-                    }
-                }
-            }
-        }
-
-        .o_form_label {
-            font-weight: $font-weight-normal;
-        }
+        --columns: 7;
```

Comment:
In these circumstances we should probably create a generic Odoo variable and assign it here.
Eg
// Primary variables:
$form-view-group-grid-size: 7;

// Form View
--formView-group-grid: var(#{$form-view-group-grid-size}):
[...]
--columns: var(--formView-group-grid);

Reply:
To solve a layout issue, I had to use the classic BS flex grid for the `o_group` element.
So this variable is not necessary anymore.